prow/repoowners/BUILD.bazel

```diff
@@ -2,13 +2,18 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 
 go_library(
     name = "go_default_library",
-    srcs = ["repoowners.go"],
+    srcs = [
+        "repoowners.go",
+        "testclient.go",
```

Comment:
Please keep test code out of the prod library (maybe put in a testing subpackage and/or add a _test.go suffix?)

Reply:
_test.go won't work because this is used elsewhere too (`lgtm` plugin). Testing subpackage is the only solution for now (eventually, repoowners.Client should be an interface).

Reply:
Hrm, subpackage doesn't work either because of circular deps and unexported fields in the client :(

Reply:
I exposed both Client and RepoOwners as interfaces and got rid of the testing code from prod. ptal